src/components/annotations/draw_arrow_head.js

```diff
@@ -35,10 +38,14 @@ var ARROWPATHS = require('./arrow_paths');
 module.exports = function drawArrowHead(el3, ends, options) {
     var el = el3.node();
     var headStyle = ARROWPATHS[options.arrowhead || 0];
+    var startHeadStyle = ARROWPATHS[options.startarrowhead || 0];
     var scale = (options.arrowwidth || 1) * options.arrowsize;
+    var startScale = (options.arrowwidth || 1) * options.startarrowsize;
     var doStart = ends.indexOf('start') >= 0;
     var doEnd = ends.indexOf('end') >= 0;
+    var doNone = ends.indexOf('none') >= 0;
```

Comment:
`'none'` is an `extras` value, so it can only be given alone - ie `ends === 'none'`

That said, I'm a little confused by the `if(do[Start|End] || doNone)` logic below - haven't we settled on being able to apply either standoff regardless of whether there's an arrowhead shown at that end (and if not, which mechanism was used to hide it)? ie that `if` statement would disappear and we always execute its block?

Reply:
Removed these `if` statements, now we always execute `end|start` blocks